R/score_forecasts.R

```diff
@@ -0,0 +1,110 @@
+#' Score forecasts
+#'
+#' @param forecasts required data.frame with forecasts in the format returned
+#' by load_forecasts
+#' @param truths required data.frame with forecasts in the format returned
+#' by load_truth
+#' @param desired_score_types character vector of scores to calculate; defaults to returning
+#' all available scores
```

Comment:
mention how valid scores are determined, maybe add link to eval_forecasts documentation